### ISSUE TITLE
Accessibility: Keyboard focus not appropriately managed on the cart page after updating address for shipping rates

### ISSUE THREAD

**Issue by amberhinds** [woothemes/woocommerce#43621]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

On the cart page in the Change Address section, the keyboard focus is not managed appropriately.
After updating the country, city, state, and zip, then pressing the update button, the number of rates found is announced by the screen reader (awesome!), but the form is collapsed and keyboard focus stays on the now hidden update button.
 

### Expected behavior

If the form is going to be collapsed, keyboard focus should be shifted to a visible element, such as the found shipping rates. container or first rate.

### Actual behavior

Focus stays on a visually hidden element.

### Steps to reproduce

1. On the cart page, expand the change address form.
2. Fill in the form and then Tab to the update button.
3. Trigger the update button with either the space bar or return/enter key.
4. Observe that there is no longer a visible focus outline on the page.

### WordPress Environment

N/A

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woo.com/products/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by senadir** [comment]
I'm unable to reproduce this, it seems after placing the form, the focus is now back to the body tag, where should focus go back to?

**Comment by senadir** [comment]
Nevermind I was able to reproduce it. I opted to return focus on the button itself, navigation can continue from there.
